swift/Tests/Addresses/BitcoinAddressTests.swift

```diff
@@ -106,7 +106,7 @@ class BitcoinAddressTests: XCTestCase {
             "tb1qrp33g0q5c5txsp9arysrx4k6zdkfs4nce4xj0gdcccefvpysxf3q0sL5k7",
             "bc1zw508d6qejxtdg4y5r3zarvaryvqyzf3du",
             "tb1qrp33g0q5c5txsp9arysrx4k6zdkfs4nce4xj0gdcccefvpysxf3pjxtptv",
-            "bc1gmk9yu",
+            "bc1gmk9yu"
```

Comment:
Swift requires to have `,` for each item, even the last one

Reply:
Codacy sees those as errors though, maybe we should update lint rules
![Screenshot from 2020-01-20 10-10-29](https://user-images.githubusercontent.com/21371810/72713374-397a1e80-3b6d-11ea-851d-f07b45a32ddf.png)